coderd/provisionerjobs.go

```diff
@@ -41,12 +77,23 @@ import (
 // @Router /organizations/{organization}/provisionerjobs [get]
 func (api *API) provisionerJobs(rw http.ResponseWriter, r *http.Request) {
 	ctx := r.Context()
+
+	jobs, ok := api.getProvisionerJobs(rw, r, nil)
+	if !ok {
+		return
```

Comment:
It's slightly confusing here that we have some code paths that write a response but others that do not. For future eyes, it might be worthwhile adding a comment regarding the meaning of the second parameter being false.

Reply:
Does [`f217a5e` (#16252)](https://github.com/coder/coder/pull/16252/commits/f217a5edbef1723a48c06413466a246fc96669bb) help?

Reply:
Perfect!